geometry_msgs/Vector3 position
geometry_msgs/Vector3 goal_position

bool[] bitmask
